regparser/layer/def_finders.py

```diff
@@ -0,0 +1,125 @@
+# vim: set fileencoding=utf-8
+"""Parsers for finding a term that's being defined within a node"""
+from itertools import chain
+import re
+
+from regparser.citations import Label
+from regparser.grammar import terms as grammar
+import settings
+
+
+class Ref(object):
+    def __init__(self, term, label, start):
+        self.term = unicode(term).lower()
+        self.label = label
+        self.start = start
+        self.end = self.start + len(term)
+        self.position = (self.start, self.end)
+
+    def __eq__(self, other):
+        """Equality depends on equality of the fields"""
+        return isinstance(other, Ref) and repr(self) == repr(other)
+
+    def __repr__(self):
+        return 'Ref( term=%s, label=%s, start=%s )' % (
+            repr(self.term), repr(self.label), repr(self.start))
+
+
+class ExplicitIncludes(object):
+    """Definitions can be explicitly included in the settings"""
```

Comment:
You probably want to expand this docstring with an example of what this does, and when and why it would want to be used.